content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1357,7 +1357,7 @@ model User {
 
 ```prisma
 model Profile {
-  authorId String @id @map("_id") @db.ObjectId
+  authorId String @id @unique @map("_id") @db.ObjectId
```

Comment:
Same here, no need for the additional @unique due to the presence of the @id

Reply:
```suggestion
  authorId String @id @map("_id") @db.ObjectId
```